lib/foreman/renderer/scope/variables/base.rb

```diff
@@ -74,13 +74,14 @@ def kickstart_attributes
             @arch      = architecture_name
             @osver     = major.try(:to_i)
             @mediapath = mediumpath(@medium_provider) if @medium_provider
-            @repos     = repos(host)
+            @additional_media = additional_media(@medium_provider) if @medium_provider
```

Comment:
```suggestion
            @additional_media = @medium_provider.nil? [] : additional_media(@medium_provider)
```

In the template you assume the variable exists, how about this ^^^ ?